client/js/components/CardEditorComponent.js

```diff
@@ -28,7 +31,7 @@ if (window.FC === undefined) { window.FC = {}; }
 
         </form>
 
```

Comment:
For card submission, add this after form?

  <hr/>
        <button onClick={() => {ReactRouter.browserHistory.goBack();}}>Done</button>